src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is started working");

```

Comment:
The message should be more informative and indicate that it is a `Bulldozer` that has started working. Consider changing the message to something like 'Bulldozer has started its work.'